L1Trigger/CSCTriggerPrimitives/src/CSCAnodeLCTProcessor.cc

```diff
@@ -171,15 +173,18 @@ void CSCAnodeLCTProcessor::checkConfigParameters() {
   CSCBaseboard::checkConfigParameters(trig_mode, max_trig_mode, def_trig_mode, "trig_mode");
   CSCBaseboard::checkConfigParameters(accel_mode, max_accel_mode, def_accel_mode, "accel_mode");
   CSCBaseboard::checkConfigParameters(l1a_window_width, max_l1a_window_width, def_l1a_window_width, "l1a_window_width");
+
+  minbx_readout_ = CSCConstants::LCT_CENTRAL_BX - l1a_window_width / 2;
```

Comment:
These are already set in the calling method `setConfigParameters`, there is no reason to redefine them here.
Perhaps it could make sense to move the `assert` here (it is a "check" in fact), provided the `setConfigParameters` is called every time after (or inside...) the constructor.

Reply:
Yes, `checkConfigParameters` is only called in the constructor or within `setConfigParameters`: [here](https://cmssdt.cern.ch/lxr/search?%21v=CMSSW_12_6_X_2022-08-28-2300&_filestring=&_string=checkConfigParameters)
Yes, I think it's sufficient to keep the assert in `checkConfigParameters`, since it's already called in the constructor.